.github/workflows/check_smoke_tests.yml

```
@@ -62,7 +62,7 @@ jobs:
         -H "Content-Type: application/json")
 
         echo "Response: $RESPONSE_1"
-        echo "$RESPONSE_1" | jq -e '.is_match == false' 
+        echo "$RESPONSE_1" | jq -e '.prediction == no_match' 
```

I think you want to compare to the string value here.
```suggestion
        echo "$RESPONSE_1" | jq -e '.prediction == "no_match"' 
```